WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardReaderType+Manual.swift

```diff
@@ -2,6 +2,7 @@ import Foundation
 import Yosemite
 
 extension CardReaderType {
+    static let allSupportedReaders: [CardReaderType] = [CardReaderType.chipper, CardReaderType.stripeM2, CardReaderType.wisepad3]
```

Comment:
Do we need this? Or can we use instead the `allCases` property of `CardReaderType`? (filtering `other` case if necessary)

Reply:
Using `allCases` includes `other`, which is empty:
![Simulator Screen Shot - iPhone 13 mini - 2022-08-17 at 09 30 00](https://user-images.githubusercontent.com/2472348/185072421-d8c5b7e0-a041-494e-96f2-e5a019e22d12.png)

Reply:
Yeah, talking with @iamgabrielma we found that we can actually return nil for .other in the `manual` property, and `compactMap` on $0.manual. That way we would filter the nils and it wouldn't be shown:

```
var manual: Manual? {
        switch self {
        case .chipper:
            return Manual(
                id: 0,
                image: .cardReaderImageBBPOSChipper,
                name: "BBPOS Chipper 2X BT",
                urlString: "https://stripe.com/files/docs/terminal/c2xbt_product_sheet.pdf"
            )
        case .stripeM2:
            return Manual(
                id: 1,
                image: .cardReaderImageM2,
                name: "Stripe Reader M2",
                urlString: "https://stripe.com/files/docs/terminal/m2_product_sheet.pdf"
            )
        case .wisepad3:
            return Manual(
                id: 2,
                image: .cardReaderImageWisepad3,
                name: "Wisepad 3",
                urlString: "https://stripe.com/files/docs/terminal/wp3_product_sheet.pdf"
            )
        case .other:
            return nil
        }
    }
```

Reply:
I think that would be a good change to make regardless of what we do with the rest of this 😊 👍

Reply:
Thanks for chiming in @joshheald ! Definitely feels better with this change, I'll add it either here or on a different PR depending on how the discussion goes and what we do with this one 👍